cc-dashboard-backend/index.js

```diff
@@ -0,0 +1,75 @@
+const functions = require('firebase-functions');
```

Comment:
While what we're building in this app project is fairly limited in scope, it can be best practice to encapsulate some of our logic into different files.  Namely, it's generally best to keep our DB access methods in a different place than our server routes.  Not only does this clean things up in terms of readabilty, but it also allows us to have a higher level of modularity.  

For example, If our DB access is isolated to a separate file and exports the method "findOneAirman" our server code can make use of it, regardless of the DBMS in use or its query language, even if we COMPLETELY overhauled the DB or replaced the DBMS - so long as the methods exported from the files providing DB interaction methods have the same name (i.e. it doesn't matter to the server if it's a PostgreSQL or MongoDB database system, since the logic of querying it is hidden behind a method which will always be called "findOneAirman" when its exported from the file that contains the DB methods).  This resembles the [Facade design pattern](https://sourcemaking.com/design_patterns/facade).